### ISSUE TITLE
Documentation versioning

### ISSUE THREAD

**Issue by brunograz** [concourse/docs#25]
## Feature request

It would be nice to have docs for different concourse versions, like a dropdown list box where the documentation for a specific concourse version can be selected. 

This would make it easier to check if certain features are available for that version. Take resource_types [params](https://github.com/concourse/docs/commit/84c02bd520a4f593bb6168fe51657763fb89fef4#diff-f074b9687aa7a72aab36d0386c35648e) as an example, it was added only in v3.7.0 and it is not available for older versions.

**Comment by jama-pivotal** [comment]
Yes! sorry, just commenting to show that we haven't forgotten about this. As we move through more versions of Concourse, i think its going to be really important to differentiate what docs are relevant to what versions of Concourse. Something we've talked about in the past is tagging our docs repo and being able to deploy each version on different subdomains, that way we can at least point folks to older docs without asking them to compile it for themselves locally!

**Comment by marco-m** [comment]
I believe this PR fixes this #152
